app/views/npq_separation/admin/applications/outcome/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= render "npq_separation/admin/applications/application_side_navigation" %>
+
+<span class="govuk-caption-m">
+  <%= @application.user.full_name %>,
+  <%= @application.course.name %>,
+  <%= @application.created_at.to_date.to_fs(:govuk_short) %>
+</span>
+
+<h1 class="govuk-heading-l">Course outcome</h1>
+
+<% all_outcomes = @application.declarations.flat_map(&:participant_outcomes) %>
```

Comment:
it doesn't look like `all_outcomes` is used.

Reply:
ah yes